packages/react-ai/src/components/AIConversation/views/Controls/FieldControl.tsx

```diff
@@ -38,14 +39,13 @@ const SendButtonBase = withBaseElementProps(Button, {
 const SendButton: typeof SendButtonBase = React.forwardRef(
   function SendButton(props, ref) {
     const { input } = React.useContext(InputContext);
-    // TODO should come from context
-    const isWaitingForResponse = false;
+    const isLoading = React.useContext(LoadingContext);
     const hasInput = !!input?.text || !!input?.files?.length;
 
     return (
       <SendButtonBase
         {...props}
-        disabled={isWaitingForResponse || !hasInput}
+        disabled={isLoading ?? !hasInput}
```

Comment:
I think this actually does need to be `||` because `??` won't use the right side if the left is false

Reply:
I guess I'm thinking if `isLoading` is `false` and `!hasInput` is `true` then don't we still want `isDisabled` to be true?

Reply:
Oh yes you are right, will update!